Extensions/import_transactions/includes/import_transactions.inc

```diff
@@ -382,7 +385,11 @@ function determinetax($taxtype, $amt) {
     $sql = "SELECT * FROM " . TB_PREF . "tax_types WHERE id=" . db_escape($taxtype);
     $result = db_query($sql, _("Could not query tax type table"));
     $row = db_fetch($result);
-    $taxper = $row['rate'];
```

Comment:
This line:
`$taxper = $row['rate'];`
should not have been removed as the `$taxper` variable is used in computation later.